server/message_hooks.go

```diff
@@ -670,8 +758,14 @@ func (p *Plugin) Delete(teamID, channelID string, user *model.User, post *model.
 	return nil
 }
 
-func (p *Plugin) DeleteChat(chatID string, user *model.User, post *model.Post) error {
-	client, err := p.GetClientForUser(user.Id)
+func (p *Plugin) DeleteChat(userID string, post *model.Post) error {
+	client, err := p.GetClientForUser(userID)
+	if err != nil {
+		p.handlePromptForConnection(userID, post.ChannelId)
```

Comment:
Let's not prompt for connection on delete -- we're only doing it when sending for now, and will circle back with messaging in these other cases differently.
```suggestion
```

Reply:
Ok, I still think can be missleading that you delete a message here and you don't know that is not really deleted from the other side.

Reply:
Done